Resources/config/services.yml

```diff
@@ -11,16 +11,21 @@ services:
             - '@assets.packages'
 
     gluggi.finder:
```

Comment:
All services are internal, but this needs to be discoverable from the controller via `$this->get()`. Some for `gluggi.assets`.

That's why they can't be private.